server/design/toolmetrics/design.go

```diff
@@ -98,6 +98,127 @@ var _ = Service("logs", func() {
 		Meta("openapi:extension:x-speakeasy-react-hook", `{"name": "ToolExecutionLogs"}`)
 	})
 
+	Method("listTelemetryLogs", func() {
+		Description("List unified telemetry logs following OpenTelemetry Logs Data Model.")
+		Security(security.ByKey, security.ProjectSlug, func() {
+			Scope("producer")
+		})
+		Security(security.ByKey, security.ProjectSlug, func() {
+			Scope("consumer")
+		})
+		Security(security.ByKey, security.ProjectSlug)
+		Security(security.Session, security.ProjectSlug)
+
+		Payload(func() {
+			Extend(ListTelemetryLogsPayload)
+			security.ByKeyPayload()
+			security.SessionPayload()
+			security.ProjectPayload()
+		})
+
+		Result(ListTelemetryLogsResult)
+
+		HTTP(func() {
+			GET("/rpc/logs.listTelemetryLogs")
+			security.ByKeyHeader()
+			security.SessionHeader()
+			security.ProjectHeader()
+
+			Param("time_start")
```

Comment:
Generally good practice to encapsulate this in something like `ListTemeletryLogsArgs` like we do in other services. These `Method()`s are already a bit dense/hard to read